terraform-hcl/README.md

```diff
@@ -1,6 +1,6 @@
-# Mondoo Terraform Action
+# Mondoo Terraform HCL Action
 
-A [GitHub Action](https://github.com/features/actions) for testing [HashiCorp Terraform](https://terraform.io) code for security misconfigurations.
+A [GitHub Action](https://github.com/features/actions) for testing [HashiCorp Terraform](https://terraform.io) code for security misconfigurations. Mondoo policies will verity [Terraform's HCL syntax]](https://www.terraform.io/language/syntax/configuration).
```

Comment:
@chris-rock first of all I believe you meant "verifies" but regardless, I am not sure that is correct. We don't really verify or validate syntax. That is more of what `terraform validate` does right? 

We might want to just remove the second sentence again, or change the description.

Reply:
Agreed.